lib/saluki-components/src/transforms/origin_enrichment/mod.rs

```diff
@@ -249,6 +249,8 @@ where
         for event in event_buffer {
             match event {
                 Event::Metric(metric) => self.enrich_metric(metric),
+                Event::EventD(_) => todo!(),
```

Comment:
Similar comment to the one above for the DogStatsD source around a mutable fallible conversion function (i.e. `Event::as_metric_mut`).